e2e/pageobjects/openshift/OcpLoginPage.ts

```diff
@@ -13,42 +13,57 @@ import { DriverHelper } from '../../utils/DriverHelper';
 import { CLASSES } from '../../inversify.types';
 import { TestConstants } from '../../TestConstants';
 import { By } from 'selenium-webdriver';
+import { Logger } from '../../utils/Logger';
 
 @injectable()
 export class OcpLoginPage {
 
     private static readonly LOGIN_PAGE_OPENSHIFT: string = 'div[class=container]';
 
     constructor(
-    @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+
+    async openLoginPageOpenShift() {
+        Logger.debug('OcpLoginPage.openLoginPageOpenShift');
```

Comment:
All such commands can be unified with `Logger.debug(this.constructor.name + "." + this.currentMethod)`, or 
```
Logger.logFqn(this);
```
```
class Logger() {
  logFqn(this) {
    Logger.debug(this.constructor.name + "." + this.currentMethod)`
  }
}
```

Reply:
Nice catch, thank you.
I have created an issue for this changes and I'll resolve it ASAP https://github.com/eclipse/che/issues/14689